web-ts/src/components/__tests__/LocationHeader.spec.tsx

```diff
@@ -59,12 +50,12 @@ describe('LocationHeader', () => {
     { code: 'en', name: 'English', path: '/augsburg/en' }
   ]
 
-  const language = 'de'
-  const city = 'augsburg'
-  const location = route => createLocation({ type: route, payload: { city, language } })
+  const languageCode = 'de'
+  const cityCode = 'augsburg'
+  const pathname = '/augsburg/de/willkommen'
   const onStickyTopChanged = (value: number) => {}
 
-  type GetByTextType = (text: string, options?: {| exact: boolean |}) => boolean
+  type GetByTextType = (text: Matcher, options?: SelectorMatcherOptions | undefined) => HTMLElement
```

Comment:
It should be enough to have either `?` or `undefined` 
```suggestion
  type GetByTextType = (text: Matcher, options?: SelectorMatcherOptions) => HTMLElement
```

Reply:
Will fix in #305